src/routes/InstancesPage/CreateInstanceModal.js

```diff
@@ -193,7 +190,7 @@ function CreateInstanceModal({
             onSelect={(_ev, selection) => onChangeAWSAccountNumber(selection)}
             placeholder="Select an AWS Account"
             isDisabled={cloudAccountIds.length <= 1}
-            popperProps={{ appendTo: 'inline' }}
+            popperProps={{ appendTo: () => document.body }}
```

Comment:
`'inline'` here caused the dropdown to be cut off inside the modal if there are many entries.